fgpyo/io/io.py

```diff
@@ -0,0 +1,175 @@
+"""
```

Comment:
I don't like having to `import fgpyo.io.io` - can these functions move into the module `__init_.py` please?